.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: "Updatecli: Dependency Management"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 */2 * * *'
```

Comment:
I think every 2h is too much. I would prefer to relax it or remove it. Ideally, we would just leave the `workflow_dispatch` and trigger this workflow manually or when the helm charts are released, at the end of https://github.com/kubewarden/helm-charts/blob/main/.github/workflows/helm-chart-release.yml#L176.

Reply:
Changed to once a day, maybe change to trigger on helm-chart,

Reply:
Thanks! resolving.